canva.com.website-v2.json

```diff
@@ -0,0 +1,55 @@
+{  
+   "providerId":"canva.com",
+   "providerName":"Canva",
+   "serviceId":"website-v2",
+   "serviceName":"Canva Websites",
+   "syncPubKeyDomain":"domainconnect.canva.com",
+   "version":1,
+   "logoUrl":"https://static.canva.com/static/images/canva_logo.svg",
+   "description":"Integrates a domain with Canva",
+   "variableDescription":"",
+   "records":[
+      {
+         "groupId":"hostingA",
+         "type":"A",
+         "host":"@",
+         "pointsTo":"%ipv4%",
+         "ttl":3600
+      },
+      {
+         "groupId":"hostingA",
+         "type":"A",
+         "host":"www",
+         "pointsTo":"%ipv4%",
+         "ttl":3600
+      },
+      {
+         "groupId":"hostingAAAA",
+         "type":"AAAA",
+         "host":"@",
+         "pointsTo":"%ipv6%",
+         "ttl":3600
+      },
+      {
+         "groupId":"hostingAAAA",
+         "type":"AAAA",
+         "host":"www",
+         "pointsTo":"%ipv6%",
+         "ttl":3600
+      },
+      {
+         "groupId":"verificationTxt",
+         "type":"TXT",
+         "host":"%verify-txt-name%",
```

Comment:
What's the point of this variable if you don't intend to use it or leave it empty?

Reply:
Hey @pawel-kow, we want to have the option of using a different name if the need arises. This way we won't need to make a new template version if we ever change our minds on the verification record format.

Reply:
ok, fair enough.